Libplanet.Headless/Hosting/LibplanetHostingExtensions.cs

```diff
@@ -15,11 +17,25 @@ public static class LibplanetServicesExtensions
     public static IServiceCollection AddLibplanet<T>(
         this IServiceCollection services,
         Configuration configuration,
-        IEnumerable<T> genesisActions)
+        IEnumerable<T> genesisActions
```

Comment:
How about adding new parameter for `Currency`s and adding [PNG](https://github.com/planetarium/planet-node/blob/5547025c57b668a726da9cad40e34623c16bbb18/PlanetNode/Currencies.cs#L8) on planet-node side in this PR?

Reply:
Could you elaborate the former one?  There's already an overload taking `nativeTokens`.  Do you mean anything else?

Reply:
Ah, I missed an overload you said. 😅 that looks enough to me.